implementation/scripts/package.sh

```diff
@@ -134,13 +140,28 @@ function buildpack::archive() {
 function buildpackage::create() {
   local output
   output="${1}"
+  buildpack_type="${2}"
+
+  util::print::title "Packaging ${buildpack_type}... ${output}"
 
-  util::print::title "Packaging buildpack..."
+  if [ "$buildpack_type" == "extension" ]; then
+    mkdir -p "${BUILD_DIR}/cnbdir"
 
-  pack \
-    buildpack package "${output}" \
-      --path "${BUILD_DIR}/buildpack.tgz" \
-      --format file
+    pushd "${BUILD_DIR}/cnbdir" > /dev/null
```

Comment:
Initially i was confused why we were not doing something similar to the `pack buildpack package --path <path>` below, then I realized that `pack extension package` doesn't have that feature.

I opened [this feature request](https://github.com/buildpacks/pack/issues/2183) on pack -if/when that lands we can clean this up.